### PR TITLE
170 add functional edit response button

### PR DESCRIPTION
**Pull request by radu-constantin**

### CODE REVIEW

File: apps/backend/src/types.ts [hunk 1]
```diff
@@ -134,11 +134,17 @@ export type CreateRequestData = Pick<Request, 'title' | 'content' | 'neighborhoo
...
 /**
- * shape of the Response data, only with the required `content` and `request_id properties
+ * shape of the Response data, only with the required `content` and `request_id` properties
  * to create a Response
  */
 export type ResponseData = Pick<Response, 'content' | 'request_id'>;
 
+/**
+ * shape of the data required to edit a response, only with the required `content` and `id` properties
+ * to edit a Response
```
Comment (nina-dk) [inline apps/backend/src/types.ts:144]: you're saying "to edit a response" twice 😁

File: apps/frontend/src/components/ResponseBox/ResponseBox.tsx [hunk 2]
```diff
@@ -98,9 +142,55 @@ export default function ResponseBox({ response, requestOwnerId }: Props) {
         </div>
         <p className={styles.createdDate}>{date}</p>
       </div>
-      <p className={styles.p}>{response.content}</p>
-      <hr className={styles.hr} />
-      <div className={styles.contact}>{contactInfo}</div>
+      {showEditForm ? (
+        <Form noValidate onSubmit={handleSubmit} role="form">
```
Comment (nina-dk) [inline apps/frontend/src/components/ResponseBox/ResponseBox.tsx:146]: Would it make sense to extract this as a separate component, or at least in a variable?

**Review by nina-dk — APPROVED** [review]
I love this!

File: apps/frontend/src/components/ResponseBox/ResponseBox.tsx (hunk 2 above)
Comment (radu-constantin) [reply apps/frontend/src/components/ResponseBox/ResponseBox.tsx:146]: I think a variable would work best!

Comment (nina-dk) [reply apps/frontend/src/components/ResponseBox/ResponseBox.tsx]: Hey I just saw this. I’ve moved it in a separate component in my branch so let me know if you’d like to move it back into this one

**Pull request merged**